### PR TITLE
Initial setup for Materialtori service

### PR DESCRIPTION
**Pull request by alihussain03**
1. Add Materialtori service setup 
2. Add Docker images for root project and Materialtori service project

### CODE REVIEW

**Comment by rvsape** [comment]
Please make a separate PR for docker related changes

**Comment by alihussain03** [comment]
i think docker changes in separate PR will not make sense as in main branch we don't have much things. for running  the docker files, we should have some running project.

**Comment by rvsape** [comment]
maybe the the docker stuff needs to wait that we have something in main

**Comment by alihussain03** [comment]
yeah maybe that could be one option. now i can remove the docker files and wait for the PR approval.

File: cc_materiaalitori_service_setup/.env
Comment (rvsape) [inline cc_materiaalitori_service_setup/.env]: .env - files should be ignored, so rename to .env-example
+ .gitignore with node_modules included

File: cc_materiaalitori_service_setup/README.md
Comment (rvsape) [inline cc_materiaalitori_service_setup/README.md]: No need for this line, since README is in the desired location already

File: cc_materiaalitori_service_setup/tsconfig.json
Comment (rvsape) [inline cc_materiaalitori_service_setup/tsconfig.json]: add "strict": true for better type checking

File: cc_materiaalitori_service_setup/README.md
Comment (rvsape) [inline cc_materiaalitori_service_setup/README.md]: eslint script does not work

File: cc_materiaalitori_service_setup/tsconfig.json
Comment (rvsape) [inline cc_materiaalitori_service_setup/tsconfig.json]: formatting

File: cc_materiaalitori_service_setup/package.json
Comment (rvsape) [inline cc_materiaalitori_service_setup/package.json]: why readline-sync?

**Comment by rvsape** [comment]
rename folder: cc-materiaalitori-service

**Comment by OpaJaa** [comment]
> rename folder: cc-materiaalitori-service

This is still missing.

**Comment by OpaJaa** [comment]
> > rename folder: cc-materiaalitori-service
> 
> This is still missing.

And actually could the folder be named cc-mt-service like cc-pk-service is named? @rvsape

**Comment by bunjovis** [comment]
@alihussain03 are you adding those changes suggested by @OpaJaa ? what about @rvsape , do you agree with changes ali has done after your review?

**Review by rvsape — CHANGES_REQUESTED** [review]
Folder name: change to cc-mt-service

**Review by OpaJaa — CHANGES_REQUESTED** [review]
Typo in the folder name cc-mt-servcie -> cc-mt-service

**Review by OpaJaa — DISMISSED** [review]

File: cc-mt-service/README.md
Comment (rvsape) [inline cc-mt-service/README.md]: directory name change

**Review by OpaJaa — DISMISSED** [review]

File: cc-mt-service/package.json
Comment (rvsape) [inline cc-mt-service/package.json]: directory name change

**Review by rvsape — CHANGES_REQUESTED** [review]
directory name change updates to README and package.json files

**Review by rvsape — APPROVED** [review]

**Review by OpaJaa — APPROVED** [review]

**Pull request merged**
